cmd/prom-migrator/main.go

```diff
@@ -154,6 +161,7 @@ func parseFlags(conf *config, args []string) {
 	flag.StringVar(&conf.writerAuth.Password, "write-auth-password", "", "Auth password for remote-write storage.")
 	flag.StringVar(&conf.writerAuth.BearerToken, "write-auth-bearer-token", "", "Bearer token for remote-write storage. "+
 		"This should be mutually exclusive with username and password.")
+	flag.BoolVar(&conf.versionPrint, "version", false, "While passing this flag to the cli, the prom-migration version will be printed at first place.")
```

Comment:
The reason why this should be treated as
https://github.com/timescale/promscale/blob/5de3a5ce6c634508bc524e32e9cad3f7470d586d/pkg/runner/args.go#L14
is because flags like `-version`, `-help` do not require any input, other than their actual presence. I agree this will set itself to true if used, but IMHO (not sure what others think), flags that do not have any input needs, should be a simple match. Plus, these flags are self-explanatory, so this saves from an explanation line as well.

Reply:
Got it, thanks for the reasoning.